MenuExample/Controller/MenuViewController.swift

```diff
@@ -19,6 +19,7 @@ class MenuViewController: UITableViewController {
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
 
+        self.view.backgroundColor = UIColor.clearColor()
```

Comment:
Thank you for explanation. Could you plz leave a TODO in code with explanation and info that we need to recheck when iOS9 will be in production. Thanks.